recipes-core/basebox-user/basebox-user.bb

```diff
@@ -10,7 +10,7 @@ inherit useradd
 
 USER = "basebox"
 # generated with openssl passwd -6 b-isdn
-PASSWORD = "$6$2zTHd7.l2MZBTq5j$LAwBkh6ILoFcqQyKhwzWe/jm.y/R3Mv0tsOhEbssq.ZLNiXivGpQUmKUWJSvoncQMj/jboLCQAH689wRfUy18."
+PASSWORD = "\$6\$2zTHd7.l2MZBTq5j\$LAwBkh6ILoFcqQyKhwzWe/jm.y/R3Mv0tsOhEbssq.ZLNiXivGpQUmKUWJSvoncQMj/jboLCQAH689wRfUy18."
```

Comment:
wouldn't singe quotes be the better (simpler/shorter/easier to understand) option here?

Reply:
The issue isn't here, the issue is then when the PASSWD_PARAMS get passed to passwd. Single quotes here won't fix anything.

Reply:
this should be part of the commit message then

Reply:
reworded slightly